google/resource_storage_bucket.go

```diff
@@ -378,3 +448,135 @@ func flattenCors(corsRules []*storage.BucketCors) []map[string]interface{} {
 	}
 	return corsRulesSchema
 }
+
+func resourceGCSBucketLifecycleCreateOrUpdate(d *schema.ResourceData, sb *storage.Bucket, update bool) error {
```

Comment:
`update` is not used as of now and may be removed. I added it initially as I thought a special treatment was required for update case as opposed to creation.